notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-989_analysis_inertia_compensation.ipynb

```diff
@@ -0,0 +1,699 @@
+{
```

Comment:
<p>Why do the ptp get larger with the inertial compensation on :sob:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>